docs/resources/aws_iam_access_keys.md

```diff
@@ -0,0 +1,164 @@
+---
+title: About the aws_iam_access_keys Resource
+---
+
+# aws_iam_access_keys
+
+Use the `aws_iam_access_keys` InSpec audit resource to test properties of all or multiple IAM Access Keys.
+
+To test properties of a Access Key, use the `aws_iam_access_key` resource.  You may also test properties of the access keys of a particular user using the `aws_iam_user` resource.
```

Comment:
To test properties of a(n) *single* Access Key, use the `aws_iam_access_key` resource.  
^^ use "an" or add "single"
To test properties of an individual user's access keys, use the `aws_iam_user` resource.
(Follow established pattern)